pkg/virt-operator/resource/apply/instancetype_test.go

```diff
@@ -20,6 +29,111 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-operator/util"
 )
 
+type fakeStrategy struct {
```

Comment:
I think this should be under its own `fake` pkg

Reply:
Added a new pkg.